### PR TITLE
issue7: bff with spring webclient

### PR DESCRIPTION
**Pull request by yangli2014**

### CODE REVIEW

File: bff_webclient/src/test/java/org/opennms/devjam2022/bff/spring/BffWebclientApplicationTests.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package org.opennms.devjam2022.bff.spring;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+class BffWebclientApplicationTests {
+
+  @Test
+  void contextLoads() {
```
Comment (denkonakov) [inline bff_webclient/src/test/java/org/opennms/devjam2022/bff/spring/BffWebclientApplicationTests.java:10]: If we don't have any tests, can we just remove the file?

File: restclient/api_v1.http [hunk 2]
```diff
@@ -0,0 +1,8 @@
+### list users
+GET http://localhost:8080/v1/users
+Accept: application/json
+
+### get user by id
+GET http://localhost:8080/v1/users/6796170878378971566
```
Comment (denkonakov) [inline restclient/api_v1.http:6]: This http files don't really make sense, right? Each time we run api-server we will regenerate data...

**Review by denkonakov — APPROVED** [review]
LGTM, but I would remove files which do not make any sense (empty tests, http files)

File: restclient/api_v1.http (hunk 2 above)
Comment (yangli2014) [reply restclient/api_v1.http:6]: This is just my debug tool

**Pull request merged**